### DISCUSSION TITLE
新手提问：simbot登录问题

### DISCUSSION THREAD

**Discussion by MoonyKun** [simple-robot/Discussions#109] [Q&A]
### 涉及的编程语言

Java

### 疑问描述

我使用的是simboot+springboot+mirai组件
但是在登录时安卓手机和平板提示版本过低
安卓手表提示扫码登录。应该怎么解决。

**Comment by ForteScarlet** [comment]
可参考 https://github.com/orgs/simple-robot/discussions/108
